bip-0301.mediawiki

```diff
@@ -32,9 +32,9 @@ However, traditional MM has two drawbacks:
 
 ==Notation and Example==
 
-Note: We use notation side:\* and main:\* in front of otherwise-ambiguous words (such as "block", "node", or "chain"), to sort the mainchain version from its sidechain counterpart. We name all sidechain users "Simon", and name all mainchain miners "Mary".
+We use notation side:\* and main:\* in front of otherwise-ambiguous words (such as "block", "node", or "chain"), to sort the mainchain version from its sidechain/alt-chain counterpart. We name all sidechain users "Simon", and name all mainchain miners "Mary".
 
-Example: imagine that a sidechain block contains 20,000 txns, each paying a $0.10 fee; therefore, the block is worth $2000 of fee-revenue. As usual: the sidechain's coinbase txn will pay this $2000 to someone (in this case, "Simon"). Under Bip301, Simon does no hashing, but instead makes one layer1 txn paying $1999 to the layer1 miners ("Mary").
+Furthermore, here is an example of BIP-301 in use. Imagine that a side:block contains 20,000 txns, each paying a $0.10 fee; therefore, the side:block is worth $2000 of fee-revenue. In BIP-301 sidechain's coinbase txn will pay this $2000 to "Simon". Simon does no hashing, but instead makes one layer1 txn paying $1999 to the layer1 miners ("Mary"). Thus, Mary ends up with all of the fee-revenue, even though she didn't do anything on the sidechain.
```

Comment:
I think I will keep "txn" here, since next to the term "coinbase" it will not confuse anyone and is shorter. But I will take the rest of the suggestions.